### PR TITLE
Variable renames for readability

### PR DESCRIPTION
**Pull request by steviez**
#### Summary of Changes
Use more descriptive variable names

### CODE REVIEW

File: perf/src/packet.rs [hunk 1]
```diff
@@ -81,37 +81,39 @@ impl PacketBatch {
...
 }
 
-pub fn to_packet_batches<T: Serialize>(xs: &[T], chunks: usize) -> Vec<PacketBatch> {
-    xs.chunks(chunks)
-        .map(|x| {
-            let mut batch = PacketBatch::with_capacity(x.len());
-            batch.packets.resize(x.len(), Packet::default());
-            for (i, packet) in x.iter().zip(batch.packets.iter_mut()) {
-                Packet::populate_packet(packet, None, i).expect("serialize request");
+pub fn to_packet_batches<T: Serialize>(data: &[T], chunk_size: usize) -> Vec<PacketBatch> {
```
Comment (jstarry) [inline perf/src/packet.rs:84]: How about `max_batch_size` instead of `chunk_size`?

File: perf/src/packet.rs [hunk 2]
```diff
@@ -81,37 +81,39 @@ impl PacketBatch {
...
+    data.chunks(chunk_size)
+        .map(|data_chunk| {
```
Comment (jstarry) [inline perf/src/packet.rs:86]: `data_chunk` sounds like a sub-unit of a single packet's data. Maybe `batch_data` or `batch_items` would be clearer

File: perf/src/packet.rs (hunk 1 above)
Comment (jstarry) [reply perf/src/packet.rs:84]: Also, `data` is usually used to describe a `&[u8]` type. Maybe could use `items`?

Comment (steviez) [reply perf/src/packet.rs]: `items` works for me; I used `chunk_size` to match the library variable name:
https://doc.rust-lang.org/src/core/slice/mod.rs.html#788

Comment (jstarry) [reply perf/src/packet.rs]: cool, sounds good!

**Pull request merged**